keyboards/xelus/xs60/soldered/config.h

```diff
@@ -0,0 +1,52 @@
+/* Copyright 2022 Harrison Chan (Xelus)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+/* Set 0 if debouncing isn't needed */
+#define DEBOUNCE 5
+
+/* Mechanical locking support. Use KC_LCAP, KC_LNUM or KC_LSCR instead in keymap */
+#define LOCKING_SUPPORT_ENABLE
+
+/* Locking resynchronize hack */
+#define LOCKING_RESYNC_ENABLE
+
+// I2C setup
+#define I2C1_SCL_PAL_MODE   4
+#define I2C1_SDA_PAL_MODE   4
+#define I2C1_TIMINGR_PRESC  0U
+#define I2C1_TIMINGR_SCLDEL 7U
+#define I2C1_TIMINGR_SDADEL 0U
+#define I2C1_TIMINGR_SCLH   45U
+#define I2C1_TIMINGR_SCLL   149U
+
+// I2C EEPROM
+#define EEPROM_I2C_24LC64
+
+// More EEPROM for layers
+#define DYNAMIC_KEYMAP_EEPROM_MAX_ADDR 8191
```

Comment:
I'm not sure that this is needed.  IIRC, the dynamic keymap feature (and via, by extension) should match whatever the eeprom driver supports.

Reply:
EEPROM definition does not include DYNAMIC_EEPROM size

in `drivers/eeprom/eeprom_i2c.h`

Reply:
No, but dynamic keymap does include the size

`platforms/eeprom.h` line 35-37, `TOTAL_EEPROM_BYTE_COUNT` is used, specifically. And that pulls `EXTERNAL_EEPROM_BYTE_COUNT` for the i2c eeprom driver, which defaults to 8192 (so the -1 in dyamanic_keymap.c ensures it's already 8191.

So, yeah, definitely not needed. 

```suggestion
```